spec/integration/v2/tailorings_spec.rb

```diff
@@ -195,6 +195,49 @@
     end
   end
 
+  path '/policies/{policy_id}/tailorings/{tailoring_id}/rule_tree' do
+    let(:policy_id) do
+      FactoryBot.create(
+        :v2_policy,
+        account: user.account,
+        profile: FactoryBot.create(:v2_profile, rule_count: 10, ref_id_suffix: 'foo', supports_minors: [1])
+      ).id
+    end
+
+    let(:item) do
+      FactoryBot.create(:v2_tailoring, :with_mixed_rules, policy: V2::Policy.find(policy_id), os_minor_version: 1)
+    end
+
+    get 'Request the Rule Tree of a Tailoring' do
+      v2_auth_header
+      tags 'Policies'
+      description 'Returns the Rule Tree of a Tailoring'
+      operationId 'TailoringRuleTree'
+      content_types
+
+      parameter name: :policy_id, in: :path, type: :string, required: true
+      parameter name: :tailoring_id, in: :path, type: :string, required: true
+
+      response '200', 'Returns the Rule Tree of a Tailoring' do
+        let(:tailoring_id) { item.id }
```

Comment:
The `item` is not reused in the context of this endpoint.

```suggestion
        let(:tailoring_id) { FactoryBot.create(:v2_tailoring, :with_mixed_rules, policy: V2::Policy.find(policy_id), os_minor_version: 1).id }
```

Reply:
What do you mean it is not reused? It IS reused...that's what I'm doing here

Reply:
Under `/policies/{policy_id}/tailorings/{tailoring_id}/rule_tree`, the `item` is not being reused, it is only used once on the line 222.

This part is unnecessary to introduce:
```
    let(:item) do
      FactoryBot.create(:v2_tailoring, :with_mixed_rules, policy: V2::Policy.find(policy_id), os_minor_version: 1)
    end
```

Reply:
No it is not, I am maintaining consistency across other integration (and non-integration) tests. The `item` variable is always used when retrieving a single entity or anything underneath.